src/foundry/client/pixi/webgl/shaders/filters/blur.d.ts

```diff
@@ -0,0 +1,165 @@
+export {};
+
+declare global {
+  /**
+   * Apply a vertical or horizontal gaussian blur going inward by using alpha as the penetrating channel.
+   */
+  class AlphaBlurFilterPass extends PIXI.Filter {
+    /**
+     * @param horizontal - If the pass is horizontal (true) or vertical (false).
+     * @param strength   - Strength of the blur (distance of sampling).
+     *                     (default: 8)
+     * @param quality    - Number of passes to generate the blur. More passes = Higher quality = Lower Perf.
+     *                     (default: 4)
+     * @param resolution - Resolution of the filter.
+     *                     (default: PIXI.Filter.defaultResolution)
+     * @param kernelSize - Number of kernels to use. More kernels = Higher quality = Lower Perf.
+     *                     (default: 5)
+     */
+    constructor(horizontal: boolean, strength?: number, quality?: number, resolution?: number, kernelSize?: number);
+
+    /**
+     * If the pass is horizontal (true) or vertical (false).
+     */
+    horizontal: boolean;
+
+    /**
+     * Strength of the blur (distance of sampling).
+     */
+    strength: number;
+
+    /**
+     * The number of passes to generate the blur.
+     */
+    passes: number;
+
+    /**
+     * The quality of the filter is defined by its number of passes.
+     */
+    get quality(): number;
+
+    set quality(value);
+
+    /**
+     * The strength of the blur filter in pixels.
+     */
+    get blur(): number;
+
+    set blur(value);
+
+    /**
+     * The kernels containing the gaussian constants.
+     */
+    static GAUSSIAN_VALUES: Record<number, number>;
```

Comment:
It doesn't really matter for such a niche class but I noticed some items don't have `@defaultValue`s which was a bit surprising to me because they seemed to be there elsewhere.

Reply:
There was a point where I just was getting through these because there's a LOT of default values and if you're this deep in the weeds you can figure out how to open the file and read the data.